packages/core-components/src/components/ErrorPanel/ErrorPanel.tsx

```diff
@@ -100,6 +100,7 @@ export const ErrorPanel = ({
 }: PropsWithChildren<ErrorPanelProps>) => {
   return (
     <WarningPanel
+      severity="error"
```

Comment:
I see that this change makes tests relying on string validation fail.

Because `Warning: My Error` now is `Error: My Error`

However, I think that a component called ErrorPanel should have the `severity=error`

wdyt , should I roll this back or fix the tests?

Reply:
Fix the tests to look for `Error: My Error` 👍